src/content/supported-icons.tsx

```diff
@@ -24,12 +24,18 @@ import { IconTerminalScreen16 } from '@hashicorp/flight-icons/svg-react/terminal
 import { IconTerraformColor16 } from '@hashicorp/flight-icons/svg-react/terraform-color-16'
 import { IconTools16 } from '@hashicorp/flight-icons/svg-react/tools-16'
 import { IconVmware16 } from '@hashicorp/flight-icons/svg-react/vmware-16'
+import { IconVaultColor16 } from '@hashicorp/flight-icons/svg-react/vault-color-16'
+import { IconBoundaryColor16 } from '@hashicorp/flight-icons/svg-react/boundary-color-16'
+import { IconConsulColor16 } from '@hashicorp/flight-icons/svg-react/consul-color-16'
+import { IconPackerColor16 } from '@hashicorp/flight-icons/svg-react/packer-color-16'
 
 export const SUPPORTED_ICONS = {
```

Comment:
We should check on the size of this component with the dictionary approach. It's likely that all these assets load every time 🤔 Definitely a polish type task.

Reply:
Yeah its definitely a tough one 🤔